adminshell/shell.py

```
@@ -0,0 +1,83 @@
+                    server_url = args[2]
+                    params = {}
+                    for arg in args[3:]:
+                        if arg.startswith('--') and '=' in arg:
+                            k, v = arg[2:].split('=', 1)
+                            params[k] = v
+                    fetch_resource(resource, server_url, params)
+            except Exception as e:
+                print(f"Error parsing fetch command: {e}")
+        else:
+            print(f"Unknown command: {command}")
+        command = input(">>> ")
```

Pylint (3.3.7)

[refactor] Too many nested blocks (6/5)

(R1702)